src/Hosting/Hosting/src/Internal/WebHost.cs

```diff
@@ -27,6 +27,7 @@
 
 namespace Microsoft.AspNetCore.Hosting
 {
+    [Obsolete]
```

Comment:
We shouldn't need to put these on internal types, that just causes more internal noise.